.github/workflows/type_completeness.yml

```diff
@@ -33,29 +33,34 @@ jobs:
           git checkout ${{ github.head_ref }}
           pip install . -U
           pyright --verifytypes telegram --ignoreexternal --outputjson > pr.json || true
+          pyright --verifytypes telegram --ignoreexternal > pr.readable || true
       - name: Compare Completeness
         uses: jannekem/run-python-script-action@v1
         with:
           script: | 
             import json
             import os
+            from pathlib import Path
 
             base = float(
               json.load(open("base.json", "rb"))["typeCompleteness"]["completenessScore"]
             )
             pr = float(
               json.load(open("pr.json", "rb"))["typeCompleteness"]["completenessScore"]
             )
+            base_text = f"After this PR, type completeness will be {round(pr, 3)}."
             if pr < (base - 0.1):
                 text = f"This PR decreases type completeness by {round(base - pr, 3)} ❌"
                 set_summary(text)
-                error(text)
+                error(f"{base_text} {text}")
```

Comment:
shouldn't this be the opposite, i.e `error(f"{text} {base_text}")` so it reads as 
```
This PR decreases type completeness by 2% ❌
After this PR, type completeness will be 98%
``` 
something like this would be shown?

Reply:
mh, I don't really care for the order … Those are two independent true statements and switching them around makes them no less true. If you prefer it as `{text} {base_text}`, I can change it.

Reply:
for me I'd like to see the total amount after I've seen how much it has been reduced (mostly cause of the wording "This PR", "After this PR")